src/styles/ant-vars.less

```diff
@@ -9,12 +9,12 @@ https://github.com/ant-design/ant-design/blob/master/components/style/themes/def
 @baby-purple: #b59ff6;
 @dark-purple: #3b3649;
 @allen-purple: #8d87aa;
-@greyish-brown: #4a4a4a;
+@grayish-brown: #4a4a4a;
```

Comment:
oh, so you're a grey with an "A" person 🧐

Reply:
Omg called out. We can be a "grey" shop as long as its all the same lol

Reply:
I looked it up and apparently `grey` is the British way, which I find offensive because that's the way I spell it. You don't need to change them back

Reply:
((Megan secretly British confirmed))